packages/client/src/Call.ts

```diff
@@ -1165,6 +1165,34 @@ export class Call {
     });
   };
 
+  /**
+   * Updates the audio output level for a given participant or for all the participants and the default call audio output level.
+   * @param level
+   * @param participant
+   */
+  setAudioOutputLevel(level: number, participant?: StreamVideoParticipant) {
```

Comment:
For consistency with the other APIs, lets use `setAudioOutputLevel = (level: number, sessionId?: string) => {}`

Reply:
Good point, adjusting.